### PR TITLE
{chem}[foss/2019b] RDKit v2020.09.3 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2656.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/ab9d6de7607e089f3385427c84488098 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3123.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/bf9d205add8e58dc221efa49c16c9633 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/r/RDKit/RDKit-2020.09.3-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+configopts = "-DPy_ENABLE_SHARED=1 -DRDK_INSTALL_STATIC_LIBS=OFF -DRDK_INSTALL_INTREE=OFF "
+configopts += "-DRDK_BUILD_INCHI_SUPPORT=ON "
+configopts += "-DBoost_INCLUDE_DIR=$EBROOTBOOST/include -DBoost_LIBRARY_DIR_RELEASE=$EBROOTBOOST/lib "
+
+# merge source directory into build directory in order to run the tests
+buildopts = '&& cp -RT %(builddir)s/%(namelower)s-*/ ./ && '
+buildopts += 'export RDBASE=$PWD && export PYTHONPATH=$PWD:$PYTHONPATH && '
+# 'ctest' allows to pass additional arguments opposed to 'make test'
+# Exclude RGroup test on intel - https://github.com/rdkit/rdkit/issues/3291
+buildopts += 'ctest --output-on-failure -E RGroup'
```
Comment (smoors) [inline easybuild/easyconfigs/r/RDKit/RDKit-2020.09.3-foss-2019b-Python-3.7.4.eb:47]: @boegel as this is a foss build, can you comment out the `-E RGroup` here?

**Review by smoors — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/69d128dc24bd48600d6e9b0202349088 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node369.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/ad3b9326fe7f5c0a4aacc72bd5c7f7a3 for a full test report.

**Comment by smoors** [comment]
@boegel failed test, which I fixed with `RDKit-2019.09.3_fix_tests.patch` for a previous version.
can you re-add that patch here?

```
AIL: testCrambin (__main__.TestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/3590926.master01.hydra.brussel.vsc/RDKit/2020.09.3/foss-2019b-Python-3.7.4/rdkit-Release_2020_09_3/Code/GraphMol/Wrap/rough_test.py", line 4012, in testCrambin
    self.assertEqual(matches, ((166, 167, 168), (298, 299, 300), (320, 321, 326)))
AssertionError: Tuples differ: ((166, 167, 168), (298, 299, 300), (320, 326, 321)) != ((166, 167, 168), (298, 299, 300), (320, 321, 326))

First differing element 2:
(320, 326, 321)
(320, 321, 326)

- ((166, 167, 168), (298, 299, 300), (320, 326, 321))
?                                             -----

+ ((166, 167, 168), (298, 299, 300), (320, 321, 326))
?                                          +++++
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3113.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/bf3a8b64b0a5645638b132fb69c08f1b for a full test report.

**Comment by boegel** [comment]
@smoors Why are the tests failing for you on Skylake, but not for me?! 🤔

**Comment by smoors** [comment]
see https://github.com/rdkit/rdkit/issues/3009
AFAIU, the tests are too strik: RDKit returns the correct 3 atoms, but there is no guarantee on the order. most of the time the order is the same but clearly not always.

**Comment by verdurin** [comment]
@boegel any update on what to do about the failing tests?

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/30db7b6876b8112e5cdc9e652d7262a5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2616.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/70d61f286d7cc4c0114f45d049fbd6d6 for a full test report.

**Comment by boegel** [comment]
@verdurin Should be good to go now!

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.6
See https://gist.github.com/299c370bc4b4c15d37a52483123dc7db for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**